src/theme.js

```diff
@@ -41,55 +43,79 @@ export default {
     },
   },
   typography: {
-    body1: {
-      color: color.deepPurple,
-      fontFamily: "Open Sans, Ariomo, Arial, sans-serif",
-      fontSize: "16px",
-      lineHeight: "22px",
-    },
-    body2: {
-      color: color.deepPurple,
-      fontFamily: "Open Sans, Ariomo, Arial, sans-serif",
-      fontSize: "14px",
-      lineHeight: "22px",
-    },
     h1: {
-      fontFamily: "Ariomo, Arial, sans-serif",
+      fontFamily: "Open Sans, sans-serif",
```

Comment:
Another wet concern. Can font-family just be set at the document/body level? Are there pre-existing styles we're overriding by specifically setting font-family at these elements?